build/salkafka/m1m3.yaml

```diff
@@ -1,95 +1,6 @@
 component: MTM1M3
 queue_len: 1000
 topic_sets:
-  - commands:
```

Comment:
Is this whole block being handled internally now? The resulting file has 13 blocks but I know the producer shows 14. What happened to all of theses?

Reply:
Yes... The producer is configured to "fill in the blanks"... so everything that is missing goes into a single process. The way it was before it was creating an additional one with any missing topic. Now I only list the ones I want to have running separately. Makes it easier to maintain.